### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by wedge22**
Updated the first two links, added notes in ## for other links that do not exist and need to be removed or updated.

**Description:** <Describe what has changed.>

**Link to Splunk idea:** <Link to Splunk idea, see https://ideas.splunk.com>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Comment by wedge22** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-* [Troubleshoot the Collector for Kubernetes containers](https://docs.splunk.com/observability/en/gdi/opentelemetry/collector-kubernetes/troubleshoot-k8s-container.html)
+* [Troubleshoot the Collector for Kubernetes](https://docs.splunk.com/observability/en/gdi/opentelemetry/collector-kubernetes/k8s-troubleshooting/troubleshoot-k8s-landing.html)
+* [Troubleshoot the Collector for Kubernetes containers](https://docs.splunk.com/observability/en/gdi/opentelemetry/troubleshooting.html)
 
-For general troubleshooting of the Splunk Distribution of the OpenTelemetry Collector see [Splunk OpenTelemetry Collector troubleshooting documentation](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/troubleshooting.md](https://docs.splunk.com/observability/en/gdi/opentelemetry/troubleshooting.html).
+##The linked page does not exist## For general troubleshooting of the Splunk Distribution of the OpenTelemetry Collector see [Splunk OpenTelemetry Collector troubleshooting documentation](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/troubleshooting.md](https://docs.splunk.com/observability/en/gdi/opentelemetry/troubleshooting.html).
```
Comment (atoulme) [inline docs/troubleshooting.md:8]:
```suggestion
For general troubleshooting of the Splunk Distribution of the OpenTelemetry Collector see [Splunk OpenTelemetry Collector troubleshooting documentation](https://docs.splunk.com/observability/en/gdi/opentelemetry/troubleshooting.html).
```

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 
-You might review the [OpenTelemetry Collector troubleshooting documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md) as well.
+##The linked page does not exist## You might review the [OpenTelemetry Collector troubleshooting documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md) as well.
```
Comment (atoulme) [inline docs/troubleshooting.md:10]:
```suggestion
```

**Comment by atoulme** [comment]
Thank you for your help! I have amended your changes to fix the links or remove the sentence.

**Review by atoulme — APPROVED** [review]

**Pull request merged**